include/drivers/can.h

```diff
@@ -703,66 +703,67 @@ static inline int can_write(const struct device *dev, const uint8_t *data, uint8
  * @retval filter_id on success.
  * @retval -ENOSPC if there are no free filters.
  */
-static inline int can_attach_isr(const struct device *dev, can_rx_callback_t callback,
-				 void *user_data, const struct zcan_filter *filter)
+static inline int can_add_rx_filter(const struct device *dev, can_rx_callback_t callback,
+				    void *user_data, const struct zcan_filter *filter)
 {
 	const struct can_driver_api *api = (const struct can_driver_api *)dev->api;
 
-	return api->attach_isr(dev, callback, user_data, filter);
+	return api->add_rx_filter(dev, callback, user_data, filter);
 }
 
 /**
  * @brief Statically define and initialize a CAN RX message queue.
  *
- * The message queue's ring buffer contains space for @a size CAN frames.
+ * The message queue's ring buffer contains space for @a nframes CAN frames.
```

Comment:
```suggestion
 * The message queue's ring buffer contains space for @a max_frames CAN frames.
```
Would be consistent with the msgq API

Reply:
Good idea, changed.